apps/src/gamelab/GameLabP5.js

```diff
@@ -451,13 +452,40 @@ GameLabP5.prototype.registerP5EventHandler = function (eventName, handler) {
   this.p5[eventName] = handler;
 };
 
+GameLabP5.prototype.setP5FrameRate = function () {
```

Comment:
Should this method be "private"? (at least annotated as such?)

Reply:
👍